IDEAS/Examples/IBPSA/SingleZoneResidentialHydronic.mo

```diff
@@ -56,12 +56,13 @@ model SingleZoneResidentialHydronic
     annotation (Placement(transformation(extent={{100,-10},{120,10}})));
   Utilities.Time.CalendarTime calTim(zerTim=IDEAS.Utilities.Time.Types.ZeroTime.NY2019)
     annotation (Placement(transformation(extent={{-100,60},{-80,80}})));
-  Modelica.Blocks.Sources.RealExpression yOcc(y=if calTim.hour > 7 and calTim.hour
-         < 20 then 1 else 0)
+  Modelica.Blocks.Sources.RealExpression yOcc(y=if (calTim.hour < 7 or calTim.hour
+         > 19) or calTim.weekDay > 5 then 1 else 0)
```

Comment:
heating day and night during the weekend?

Reply:
This would rather mean that the occupant has not much social life and spends the entire weekend at home. The heating is continuously working but only because the emission pump input type, boiler supply temperature and TRV setpoints are constant.

Reply:
https://github.com/open-ideas/IDEAS/pull/1111/commits/bd996382ee551772ffb685329a6141b4f9aa7714

Reply:
thx!